src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
 
     ###### SoilCanopyModel ######
 
+    ## stored in p (diagnostics variables stored in the cache) ##
+
+    ## Canopy Module ##
+	
+	### Canopy - Solar Induced Fluorescence 
+	    # Solar Induced Fluorescence
     add_diagnostic_variable!(
-        short_name = "slw",
-        long_name = "Soil Liquid Water",
-        standard_name = "soil_liquid_water",
-        units = "m^3 m^-3",
-        comments = "Soil moisture, the liquid water volume per soil volume. This liquid water is located in the soil pores.",
+        short_name = "sif",
+        long_name = "Solar Induced Fluorescence",
+        standard_name = "solar_induced_fluorescence",
+        units = "W m^-2",
+        comments = "The fluorescence of leaves induced by solar radiation. This quantity is correlated with photosynthesis activity.",
         compute! = (out, Y, p, t) ->
-            compute_soil_water_liquid!(out, Y, p, t, land_model),
+            compute_solar_induced_fluorescence!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Autotrophic respiration
+        # Autotrophic respiration
     add_diagnostic_variable!(
-        short_name = "infil",
-        long_name = "Infiltration",
-        standard_name = "infiltration",
-        units = "m s^-1", # double check
-        comments = "The flux of liquid water into the soil.",
+        short_name = "ra",
+        long_name = "Autotrophic Respiration",
+        standard_name = "autotrophic_respiration",
+        units = "mol CO2 m^-2 s^-1", # note throughout: make sure what we express as kg or mol, CO2 or C
```

Comment:
I think this may be kg of C.
I think we have a bug in the units in the code. Maybe worth fixing in this PR?
https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Vegetation/canopy_parameterizations.jl#L1004 [maintenance is in kg/m^2/s]
but here it is used with An: https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Vegetation/canopy_parameterizations.jl#L1040
and An has units of mol CO2.

Reply:
I think it would be good to decide together what to do with flux of CO2 @braghiere , do we express them is `mol CO2`, `kg C`, or other? 
These are all SI, so we just have to pick one. 
Usually, in the literature, we use mol CO2 for fluxes (umol), and kg C for budgets (change in pool size or pool size). 
I would vote for mol CO2, but let's all agree.

Reply:
Yep. Carbon fluxes is usually in umolCO2.m-2.s-1 (or kgCm-2-day-1) and pools are in kgC.